docs/output.md

```diff
@@ -65,18 +65,18 @@ osv-scanner --format markdown your/project/dir
 **Raw output:**
 
 ```
-| OSV URL | CVSS | Ecosystem | Package | Version | Source |
-| --- | --- | --- | --- | --- | --- |
-| https://osv.dev/GHSA-c3h9-896r-86jm<br/>https://osv.dev/GO-2021-0053 | 8.6 | Go | github.com/gogo/protobuf | 1.3.1 | ../scorecard-check-osv-e2e/go.mod |
-| https://osv.dev/GHSA-m5pq-gvj9-9vr8<br/>https://osv.dev/RUSTSEC-2022-0013 | 7.5 | crates.io | regex | 1.5.1 | ../scorecard-check-osv-e2e/sub-rust-project/Cargo.lock |
+| OSV URL                                                                   | CVSS | Ecosystem | Package                  | Version | Source                                                 |
```

Comment:
So adding space here is helpful, as it prints raw, so it will be easier for the user to read. However, I don't really think you need to update the white space of the tables that get rendered. It isn't wrong, but I don't think it is necessary. I also think it degrades over time. Anyone who makes changes in the future may feel inclined to make the lines "work" but they may or may not do it.

Reply:
Hmm... this is automatically added by vscode when saving, I'll have a look at disabling it.

Reply:
Undid this change in the output.md file